File: jdroid-android-about/src/debug/java/com/jdroid/android/about/AboutDebugAppLifecycleCallback.java
package com.jdroid.android.about;

import android.content.Context;

import com.jdroid.android.debug.DebugSettingsHelper;
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback;

public class AboutDebugAppLifecycleCallback extends ApplicationLifecycleCallback {

	@Override
	public void onProviderInit(Context context) {
		DebugSettingsHelper.addPreferencesAppender(new RateAppDebugPrefsAppender());
	}
}

File: jdroid-android-about/src/debug/java/com/jdroid/android/about/AboutDebugAppLifecycleCallback.kt
package com.jdroid.android.about

import android.content.Context

import com.jdroid.android.debug.DebugSettingsHelper
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback

class AboutDebugAppLifecycleCallback : ApplicationLifecycleCallback() {

    override fun onProviderInit(context: Context) {
        DebugSettingsHelper.addPreferencesAppender(RateAppDebugPrefsAppender())
    }
}
